Xamarin.Essentials/AppInfo/AppInfo.ios.tvos.watchos.mac.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
```

Comment:
Nitpick: it's `macOS`, not `mac`, which means the filename should be `AppInfo.ios.tvos.watchos.macos.mac`

Reply:
hehe, done. Especially for you @rolfbjarne.